src/test/java/org/entando/kubernetes/service/EntandoBundleUtilitiesTest.java

```diff
@@ -19,7 +27,7 @@ public void shouldReturnVersionDirectly() {
 
     @Test
     public void shouldThrowAnErrorAsVersionIsNotDefined() {
-        Exception ex = Assertions.assertThrows(RuntimeException.class, () -> {
+        Exception ex = assertThrows(RuntimeException.class, () -> {
```

Comment:
@firegloves assert should check for EntandoComponentManagerException, also see the code smell related to this part